### PR TITLE
fix: edge case where pubs with sibling edges of different domains would 500

### PR DESCRIPTION
**Pull request by tefkah**
- fix: include community in pub edge includes in for sibling edges to prevent weird connection 500 edge case
- fix: revert base hostname for pubdoc to be req.hostname as that has been proven to work

### CODE REVIEW

**Review by gabestein — APPROVED** [review]
Thanks! Looks good.

**Review by gabestein — CHANGES_REQUESTED** [review]
Found a small issue here that prevents merging, unfortunately. On custom domain pages with any connections at all, it will set the surrogate-key to the community subdomain rather than the hostname, which will prevent purging.

E.g., on `https://duqduqdomaintest.underlay.org/pub/zkme9ef3/release/9`, surrogate-key is gabestein.duqduq.org rather than `duqduqdomaintest.underlay.org`

**Comment by tefkah** [comment]
Hmm i think this should work, I reverted this problem from the previous commit. AFAIK the one on duqduq is the previous one right?
This PR has the same behavior in that regard as the one on prod rn

File: server/routes/pubDocument.tsx [hunk 1]
```diff
@@ -207,15 +207,18 @@ const setSurrogateKeysHeadersForPubEdges = async (
...
+
+			if (acc.includes(hostname)) {
+				return acc;
+			}
 
 			acc.push(hostname);
 			return acc;
 		},
-		[getCorrectHostname(subdomain, domain)] as string[],
+		[req.hostname] as string[],
```
Comment (tefkah) [inline server/routes/pubDocument.tsx:221]: i mean this change here: above is how it is on duqduq, below is how it is on prod

**Review by gabestein — APPROVED** [review]

**Pull request merged**